atom/browser/api/atom_api_browser_window.h

```diff
@@ -103,6 +105,8 @@ class BrowserWindow : public TopLevelWindow,
   // it should be cancelled when we can prove that the window is responsive.
   base::CancelableClosure window_unresponsive_closure_;
 
+  std::string background_color;
```

Comment:
what about the default value?

Reply:
What should the default value be?

Reply:
We're probably better off adding a `GetBackgroundColor()` method to `TopLevelWindow` and querying that here -- that way we wouldn't have to wonder what the default value should be.